pootle/apps/accounts/models.py

```diff
@@ -186,29 +186,41 @@ def top_scorers(cls, days=30, language=None, project=None, limit=5):
         past = now + datetime.timedelta(-days)
 
         lookup_kwargs = {
-            'scorelog__creation_time__range': [past, now],
+            'creation_time__range': [past, now],
         }
 
         if language is not None:
             lookup_kwargs.update({
-                'scorelog__submission__translation_project__language__code':
+                'submission__translation_project__language__code':
                     language,
             })
 
         if project is not None:
             lookup_kwargs.update({
-                'scorelog__submission__translation_project__project__code':
+                'submission__translation_project__project__code':
                     project,
             })
 
-        top_scorers = cls.objects.hide_meta().filter(
+        meta_user_ids = cls.objects.meta_users().values_list('id', flat=True)
+        top_scores = ScoreLog.objects.values("user").filter(
             **lookup_kwargs
-        ).annotate(
-            total_score=Sum('scorelog__score_delta'),
+        ).exclude(user__pk__in=meta_user_ids).annotate(
+            total_score=Sum('score_delta'),
         ).order_by('-total_score')
 
         if isinstance(limit, (int, long)) and limit > 0:
-            top_scorers = top_scorers[:limit]
+            top_scores = top_scores[:limit]
+
+        users = dict(
+            (user.id, user)
+            for user in cls.objects.filter(
+                pk__in=[item['user'] for item in top_scores]))
```

Comment:
Not planning to bike-shed here: it brings my attention the recent trend not to dedent/align vertically parens/braces/brackets as we have always done, and I'm worried this creates a mess in the code which is detrimental to the reader's eyes. In some cases it also contributes to diff noise.

Unfortunately I cannot be more constructive than calling everyone to please respect the style which we have always used and being consistent with it, since I'm not aware of any automated tools that can take care of Python formatting in a fine-grained way (yapf might be close).